docs/lib/restapi/fragments/flutter/getting-started.md

```diff
@@ -0,0 +1,52 @@
+The Amplify CLI deploys REST APIs and handlers using [Amazon API Gateway](http://docs.aws.amazon.com/apigateway/latest/developerguide/) and [AWS Lambda](http://docs.aws.amazon.com/lambda/latest/dg/).
+
+The API category will perform SDK code generation which when used with the `AWSMobileClient` can be used for creating signed requests for Amazon API Gateway when the service Authorization is set to `AWS_IAM` or when using a [Cognito User Pools Authorizer](https://docs.aws.amazon.com/apigateway/latest/developerguide/apigateway-integrate-with-cognito.html).
+
+See the authentication section for more details for using the `AWSMobileClient` in your application.
```

Comment:
Don't think this is valid for flutter.

Reply:
duplicate of above issue

Reply:
What's the resolution here? I still see `AWSMobileClient` in the flutter docs.

Reply:
@Amplifiyer this file only exists for SDK part and not for Amplify - my bad for including it.  We can ignore the contents of getting-started.md as we don't need to include it (it's been deleted).